File: Java_20Q/Q14.java
import java.util.Scanner;

public class Q14 {
  public static void Euclid(int num1, int num2) {

    int r = num1 % num2;

    while (r > 0) {
      num1 = num2;
      num2 = r;
      r = num1 % num2;
    }
    System.out.println("GCD: " + num2);
  }

  public static void main(String[] args) {

    int num1;
    int num2;

    Scanner n = new Scanner(System.in);

    System.out.println("Enter first number: ");

    num1 = n.nextInt();

    System.out.println("Enter Second number: ");

    num2 = n.nextInt();

    Euclid(num1, num2);

  }
}

File: C++_20Q/Q14.cpp
#include <iostream>

int main() {

  int num1;
  int num2;
  int r;

  std::cout << "Enter first number: " << std::endl;
  std::cin >> num1;

  std::cout << "Enter second number: " << std::endl;
  std::cin >> num2;

  r = num1 % num2;
  while (r > 0) {
    std::cout << "Entered Loop!" << std::endl;
    num1 = num2;
    num2 = r;
    r = num1 % num2;
  }
  std::cout << "The GCD: " << num2 << std::endl;
}
